ding/model/template/efficientzero/efficientzero_tictactoe_model.py

```diff
@@ -0,0 +1,607 @@
+"""
+The following code is adapted from https://github.com/YeWR/EfficientZero/blob/main/config/atari/model.py
+"""
+
+import math
+import torch
+
+import numpy as np
+import torch.nn as nn
+from ding.utils import MODEL_REGISTRY
+from ding.model.template.efficientzero.efficientzero_base_model import BaseNet, renormalize
+
+
+class DiscreteSupport(object):
+
+    def __init__(self, min: int, max: int, delta=1.):
+        assert min < max
+        self.min = min
+        self.max = max
+        self.range = np.arange(min, max + 1, delta)
+        self.size = len(self.range)
+        self.delta = delta
+
+
+value_support = DiscreteSupport(-10, 10, delta=1)
+reward_support = DiscreteSupport(-10, 10, delta=1)
+
+
+def inverse_reward_transform(reward_logits):
+    return inverse_scalar_transform(reward_logits, reward_support)
+    # return reward_logits  # TODO
+
+
+def inverse_value_transform(value_logits):
+    return inverse_scalar_transform(value_logits, value_support)
+    # return value_logits  # TODO
+
+
+def inverse_scalar_transform(logits, scalar_support):
```

Comment:
can we divide this function into two parts: `C51Head` in model and `value_rescale` function in policy